docs/guides/typescript-transformers.mdx

```diff
@@ -10,6 +10,17 @@ Some transformer plugins which are made for vanilla TypeScript also work for rob
 
 [You can find a list of transformer plugins for vanilla TypeScript here.](https://github.com/cevek/ttypescript#transformers)
 
+## Post installation
+
+To install a transformer plugin, you have to update your tsconfig.json.
+```json
+"compilerOptions": {
+	"plugins": [
+		"transform": "transformer-plugin-name"
```

Comment:
This is a syntax error - the correct form would be `{ "transform": "transformer-plugin-name" }`

Reply:
JSON has been used like this before in other pages.